test/check/check.go

```diff
@@ -157,7 +157,10 @@ func CheckForLeftoverObjects(currentVersion string) {
 	secrets := corev1.SecretList{}
 	err = framework.Global.Client.List(context.Background(), &listOptions, &secrets)
 	Expect(err).NotTo(HaveOccurred())
-	Expect(secrets.Items).To(BeEmpty(), "Found leftover objects from the previous operator version")
+	// TODO: remove this after fixing https://github.com/kubevirt/cluster-network-addons-operator/issues/190
+	for _, secret := range secrets.Items {
+		Expect(secret.Name).ToNot(Equal("kubemacpool-webhook-secret"), "Found leftover objects from the previous operator version")
```

Comment:
I'm sorry I missed this one the last time. With your change we don't report which object has been unexpectedly found (unlike the original code). Could you fix it please? I say first drop all KubeMacPool objects from the list and then check if empty.

Reply:
done